### PR TITLE
Add integration service and migrate crisp to it

### PR DESCRIPTION
**Pull request by cosmosgenius**

### CODE REVIEW

File: app/templates/activate.hbs [hunk 1]
```diff
@@ -1,12 +0,0 @@
-{{page-title "Activate"}}
-<div class="auth-main-container">
-  <AuthAssets></AuthAssets>
-  <div class="activate-container">
-    <div class="activate-main">
```
Comment (Praseetha-KR) [inline app/templates/activate.hbs:5]: can you remove `activate-container` & `activate-main` style definitions if not used

File: app/utils/install-crisp.js [hunk 2]
```diff
@@ -1,21 +0,0 @@
-// jshint ignore: start
-import ENV from 'irene/config/environment';
-
-const installCrisp = function () {
-  if (!ENV.crispWebsiteId || ENV.crispWebsiteId === '') {
-    // eslint-disable-next-line no-console
-    return console.log("Crisp Disabled");
-  }
-  window.$crisp = [];
-  window.$crisp.push(['do', 'chat:hide']);
```
Comment (Praseetha-KR) [inline app/utils/install-crisp.js:10]: Can you verify locally if chat container is hidden by default?

**Review by Praseetha-KR — CHANGES_REQUESTED** [review]

File: app/templates/activate.hbs (hunk 1 above)
Comment (cosmosgenius) [reply app/templates/activate.hbs:5]: removed

File: app/utils/install-crisp.js (hunk 2 above)
Comment (cosmosgenius) [reply app/utils/install-crisp.js:10]: default is hidden

**Review by Praseetha-KR — APPROVED** [review]

**Pull request merged**